istioctl/pkg/tag/generate.go

```diff
@@ -149,6 +151,29 @@ func Generate(ctx context.Context, client kube.Client, opts *GenerateOptions, is
 	return tagWhYAML, nil
 }
 
+func fixWhFailurePolicy(client kube.Client, vwc *admitv1.ValidatingWebhookConfiguration) (*admitv1.ValidatingWebhookConfiguration, error) {
+	var curWebhooks []admitv1.ValidatingWebhook
+	curObj, err := client.Kube().AdmissionregistrationV1().ValidatingWebhookConfigurations().
```

Comment:
We already get the webhook on L104, and we even have `fixWhConfig` on L140, can we reuse those?

Reply:
The webhook on L104 is `MutatingWebhookConfiguration` and the one actually need to be fixed is `ValidatingWebhookConfiguration`, I think we still need to fetch the in-cluster VWC to do the comparision?

Reply:
I think putting logics into the `fixWhConfig` is a great idea